doc/docs.md

```diff
@@ -1025,6 +1030,8 @@ posts_repo := new_repo<Post>(db)
 user := users_repo.find_by_id(1)?
 post := posts_repo.find_by_id(1)?
 ```
+Generics enables you to write flexible, reusable functions and types that can work with any type.
+generics method declarations have a type parameter section delimited by angle brackets `<T>` that precedes the method's return type.
```

Comment:
"generics" should be "Generic".  Capitalize at the beginning of a sentence, and singular to match the sentence intent.